apps/worker/src/request_processor/request.processor.service.ts

```diff
@@ -39,6 +39,7 @@ export class RequestProcessorService extends BaseConsumer {
     try {
       const dsnpUserId: MessageSourceId = this.blockchainService.api.registry.createType('MessageSourceId', job.data.dsnpId);
       const providerId: ProviderId = this.blockchainService.api.registry.createType('ProviderId', job.data.providerId);
+      this.graphStateManager.removeUserGraph(dsnpUserId.toString());
```

Comment:
before a change request is process its ideal to clean the user from state so we can set a fresh state